salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -113,6 +115,47 @@ Wait minion available:
 
 {%- endif %}
 
+{%- if 'etcd' in roles and 'etcd' not in skip_roles %}
+
+Prepare etcd dependencies:
+  salt.state:
+    - tgt: {{ node_name }}
+    - saltenv: metalk8s-{{ version }}
+    - sls:
+      - metalk8s.roles.internal.node-without-calico
+      - metalk8s.kubernetes.etcd.prepared
+
+Register the node into etcd cluster:
+  salt.runner:
+    - name: state.orchestrate
+    - pillar: {{ pillar | json  }}
+    - mods:
+      - metalk8s.orchestrate.register_etcd
+    - require:
+      - salt: Prepare etcd dependencies
+
+Install and wait for healthy etcd member:
+  salt.state:
+    - tgt: {{ node_name }}
+    - saltenv: metalk8s-{{ version }}
+    - timeout: 120
+    - sls: metalk8s.kubernetes.etcd.installed
+    - require:
+      - salt: Register the node into etcd cluster
```

Comment:
Since register break the etcd cluster from 1 to 2 IMO no point to register before installing etcd.
I suggest we register just after `etcd.installed` then we shouldn't get the error we have before as `metalk8s.kubernetes.etcd.installed` will basically only render one file